docs/design/autoware-interfaces/ad-api/list/api/planning/index.md

```diff
@@ -0,0 +1,96 @@
+After this, when the vehicle tries to start (is still stopped), the state will be STARTING.
+In this state, calling the start API changes the state to MOVING and the vehicle starts.
+This mechanism can add processing such as announcements before the vehicle starts.
+Depending on the configuration, the state may transition directly from STOPPED to MOVING.
+
+![planning-state](./docs/state.drawio.svg)
+
+| State    | Description                                     |
+| -------- | ----------------------------------------------- |
+| STOPPED  | The vehicle is stopped.                         |
+| STARTING | The vehicle is stopped, but is trying to start. |
+| MOVING   | The vehicle is moving.                          |
+
+## Factors
+
+The planning factors is an array of information on the behavior that the vehicle plans.
+Each factor has type shown below, pose, distance from the vehicle head to that pose, status, and detailed data depending on its type.
+See the description of each behavior below for pose and status. And see the description of factor types for detailed data.
+
+![planning-factors](./docs/factors.drawio.svg)
+
+| Factor Type                 | Behavior         | Description                                            |
+| --------------------------- | ---------------- | ------------------------------------------------------ |
+| SURROUND_OBSTACLE           | immediate stop   | There are obstacles nearby.                            |
+| FRONT_OBSTACLE              | planned stop     | There are obstacles ahead.                             |
+| INTERSECTION                | planned stop     | There are obstacles in other lanes in the path.        |
+| CROSSWALK                   | planned stop     | There are obstacles on the crosswalk.                  |
+| REAR_CHECK                  | planned stop     | There are obstacles behind like shoulder checks.       |
+| USER_DEFINED_DETECTION_AREA | planned stop     | There are obstacles in the predefined detection area.  |
+| NO_STOPPING_AREA            | planned stop     | There is not enough space beyond the no stopping area. |
+| STOP_SIGN                   | planned stop     | A stop by a stop sign.                                 |
+| TRAFFIC_SIGNAL              | planned stop     | A stop by a traffic signal.                            |
+| V2I_GATE_CONTROL            | planned stop     | A stop by a V2I gate.                                  |
+| FROM_PRIVATE_ROAD           | planned stop     | A stop before merging from private area.               |
+| SIDEWALK                    | planned stop     | A stop before crossing the sidewalk.                   |
+| LANE_CHANGE                 | lane change      | A lane change.                                         |
+| AVOIDANCE1                  | lane change      | A lane change to avoid obstacles when avoiding.        |
+| AVOIDANCE2                  | lane change      | A lane change to avoid obstacles when returning.       |
```

Comment:
If the aim is for the factor names to be understandable, we need to change AVOIDANCE1 and AVOIDANCE2 to be more clear. In the meantime, I have suggested alternative descriptions for these two factors

```suggestion
| AVOIDANCE1                  | lane change      | A lane change to avoid an obstacle in the current lane.        |
| AVOIDANCE2                  | lane change      | A lane change to return to the original lane after avoiding an obstacle.       |
```